mipi.py

```diff
@@ -218,6 +218,11 @@ def _generate_peripheral(t: Transaction, payload: bytes, options: Options) -> st
 	else:
 		raise ValueError(t)
 
+def _generate_dsc(t: Transaction, payload: bytes, options: Options) -> str:
+	if t != Transaction.DCS_COMPRESSION_MODE:
+		raise ValueError(t)
+	
+	return _generate_checked_call('mipi_dsi_compression_mode', ['dsi', 'true'], t.description)
```

Comment:
This is going to conflict with upstream https://github.com/msm8916-mainline/linux-mdss-dsi-panel-driver-generator/commit/7cb9fb2de0fbdff6342295aab65e55dd38f4a155, so I'll have to drop this when inevitably being forced to rebase for a successful upstream merge to take place :grin: